feature/client/src/main/java/com/mifos/feature/client/clientDetails/domain/repository/ClientDetailsRepository.kt

```diff
@@ -0,0 +1,22 @@
+package com.mifos.feature.client.clientDetails.domain.repository
+
+import com.mifos.core.objects.accounts.ClientAccounts
+import com.mifos.core.objects.client.Client
+import okhttp3.MultipartBody
+import okhttp3.ResponseBody
+import rx.Observable
+
+/**
+ * Created by Aditya Gupta on 06/08/23.
+ */
+interface ClientDetailsRepository {
```

Comment:
This should be in data module. feature module doesn't need to know from where data is coming. They only care about consuming the data.

Reply:
I did